### PR TITLE
Make `authorized_keys` field a text area

### PR DESCRIPTION
**Pull request by veleek**
# Change Summary
This change was originally only improving the `authorized_keys` input stuff, but in the process of making it layout properly I ended up doing a bunch of QML cleanup and switching stuff around to what seem to be more standard best practices (as best as I can tell from the QT Quick documentation.

## Make `authorized_keys` a text area instead of a single line field

Currently the `authorized_keys` field is a single line text field in line with a long-ish description.  This makes it difficult to understand what is actually supposed to go in this field in addition to making it very difficult to input multi-line values properly.
I've change this to a TextArea and moved some stuff around and updated descriptions to make this more clear.

*Note: It seems to be very confusing for some what this field actually does.  Even though I've had a raspberry pi setup for years with SSH key auth only, I had no idea what I was supposed to put here or how.  Putting a key in the incorrect format doesn't give any warnings or suggestions.  If I wanted to document this, would it be reasonable to do so inline here, or should that just be linked elsewhere?*

What's the general process for translations.  I made some changes to various fields so the translations would no longer be accurate.

## General QML Cleanup

* Removed rows property on main page, no effect for default LeftToRight layout (only applies for TopToBottom).
* Moved langbar RowLayout to be content of the background rectangle instead of a sibling inside of a parent ColumnLayout (this fixes a runtime error about anchors being used in layout).
* Enabled toggling the lang by on and off instead of just on.
* Removed some manual layout (like calculating offsets to center popup
* Removed redundant font. Family property and set font. Family in root popup.
* Switched popup to ColumnLayout to remove need to do a bunch of anchoring to get stuff layoud out.
* Move popup "title bar" and close button inside of the rectangle that defines the background.
* Tried to use layouts and anchors more consistently in ScrollView to ensure that content fills available space and wraps properly.
* Learned that QML is a pain in the butt
* `Layout.bottomMargin` property for PersistentSettings group box is set to 40 because that's the ONLY way I can get it to display a proper margin INSIDE the scroll view.
* Removed unnecessary text element.

*Original*
image

*New - in general it should be relatively unchanged (other than public keys stuff) but's it's setup better to scale and stuff if necessary.*
image

## Instructions for building without creating a package on Debian

Building and installing a Debian package everytime you make a change is a pain (especially when just working with layout changes).  Updated the Readme to include instructions on building and running on Debian without using the package.

### CODE REVIEW

**Comment by lurch** [comment]
Links to #316 ?

File: src/main.qml [hunk 1]
```diff
@@ -270,75 +269,77 @@ ApplicationWindow {
                     text: qsTr("Keyboard navigation: <tab> navigate to next button <space> press button/select item <arrow up/down> go up/down in lists")
                 }
 
-                RowLayout {
+                Rectangle {
```
Comment (veleek) [inline src/main.qml:272]: This section looks MUCH more complicated in the GitHub diff than it actually is.  If you use VSCode (or the Github.dev online one), you'll see that it's just a minor move.

File: src/OptionsPopup.qml [hunk 2]
```diff
@@ -29,414 +27,431 @@ Popup {
     property string cloudinitwrite
     property string cloudinitnetwork
 
-    // background of title
```
Comment (veleek) [inline src/OptionsPopup.qml:32]: Ditto for this file, use the Github.dev editor to view the diff and you'll see that the changes are much smaller.

**Comment by maxnet** [comment]
Implemented as textarea in 1.8

**Comment by veleek** [comment]
@maxnet - there were a number of other quality of life fixes included in this change.  Are those included as well or do I need to update this to apply to the head again?

**Pull request closed without merging**